.gitignore

```diff
@@ -11,3 +11,4 @@ src/test/resources/cloudant-2.properties
 *.iml
 build/
 .gradle/
+gradle.properties
```

Comment:
I don't think we should be ignoring the global gradle properties, local.properties we should ignore though